testing/test_deploy_app.py

```diff
@@ -67,15 +71,40 @@ def make_deploy_call(args):
     "Project": args.project,
     "ProjectNumber": args.project_number,
     # service account client id of account: kubeflow-testing@kubeflow-ci.iam.gserviceaccount.com
-    "SAClientId": "111670663612681935351",
+    "SAClientId": args.sa_client_id,
     "Token": access_token,
     "Zone": args.zone
   }
-  resp = requests.post("http://kubeflow-controller.%s.svc.cluster.local:8080/kfctl/e2eDeploy" % args.namespace, json=req_data)
+
+def make_e2e_call(args):
+  req_data = prepare_request_data(args)
+  resp = requests.post("http://kubeflow-controller.%s.svc.cluster.local:8080/kfctl/e2eDeploy" % args.namespace,
```

Comment:
Why are you contactig a locally running service? Shouldn't the prober be running in a different project and using the endpoint
deploy.kubeflow.cloud
or
deploy-staging.kubeflow.cloud

Reply:
Oh this is used for the E2E test.